src/components/BookingModal/BookingModalForm/container.js

```diff
@@ -28,48 +29,69 @@ const Container = Wrapped =>
       bookingDuration: PT.number,
       createEvent: PT.func.isRequired,
       updateEvent: PT.func.isRequired,
-      isSameDay: PT.bool.isRequired,
+      toggleRejectionMessageView: PT.bool.isRequired,
+      onFormUpdate: PT.func.isRequired,
+      hasAvailableDays: PT.bool.isRequired,
+      totalHolidays: PT.number.isRequired,
+      availableDays: PT.number.isRequired,
+      workingFromHomeBooking: PT.bool.isRequired,
     };
-
     static defaultProps = {
       isEventBeingUpdated: false,
       bookingDuration: 1,
     };
-
     constructor(props) {
       super(props);
-      const { isSameDay } = this.props;
+      const { workingFromHomeBooking } = this.props;
+
       this.state = {
         formData: {
           end: moment(),
-          eventTypeId: isSameDay ? 2 : 1 ,
+          eventTypeId: workingFromHomeBooking ? 2 : 1,
```

Comment:
Are there constants available that we can make use of here?

Reply:
There are in `utilities/eventTypes.js` @mcromie92